### PR TITLE
Update versions, delete abandoned, fix issues

### PR DESCRIPTION
**Pull request by Drsheppard01**

### CODE REVIEW

**Review by Drsheppard01 — COMMENTED** [review]
zrythm now packaged only zip/flatpak

File: pkgs.json [hunk 1]
```diff
@@ -160,6 +165,11 @@
         "description": "Ferdium is a desktop app that helps you organize how you use your favourite apps by combining them into one application.",
         "url": "https://github.com/ferdium/ferdium-app/releases/download/v6.2.4/Ferdium-linux-Portable-6.2.4-x86_64.AppImage"
     }, {
+        "name": "filen",
+        "version": "2.0.16",
+        "description": "Zero knowledge end-to-end encrypted cloud storage.",
+        "url": " https://cdn.filen.io/desktop/release/filen_x86_64.AppImage
```
Comment (blueOkiris) [inline pkgs.json:171]: I think you're missing an end `"` here

File: pkgs.json [hunk 2]
```diff
@@ -204,6 +224,12 @@
         "version": "2.7.0",
         "description": "A Native GOG and Epic Games Launcher for Linux, Windows and Mac.",
         "url": "https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/v2.7.0/Heroic-2.7.0.AppImage"
+    }, {
+        "name": "hypertrader",
+        "version": "2.36.0",
+        "description": "Crypto trading terminal with smart orders.",
+        "url": "https://download.hypertrader.app/HyperTrader-2.36.0.AppImage"
+
```
Comment (blueOkiris) [inline pkgs.json:232]: Can you delete this extra line?

**Review by blueOkiris — CHANGES_REQUESTED** [review]
There's an extra line and a missing quote, but the rest looks good to me

File: pkgs.json (hunk 1 above)
Comment (Drsheppard01) [reply pkgs.json:171]: yep, i fix this later

**Review by blueOkiris — APPROVED** [review]

**Pull request merged**